libnavigation-base/src/test/java/com/mapbox/navigation/base/internal/route/NavigationRouteExTest.kt

```diff
@@ -606,3 +612,18 @@ private fun createNavigationRouteFromResource(
     TestSDKRouteParser(),
     com.mapbox.navigation.base.route.RouterOrigin.Offboard
 ).first()
+
+private fun createIncident(startIndex: Int?, endIndex: Int?): Incident =
+    Incident.builder()
+        .id("id")
+        .geometryIndexStart(startIndex)
+        .geometryIndexEnd(endIndex)
+        .build()
```

Comment:
it would be nice to add start and end index to existing factory: `com.mapbox.navigation.testing.factories.DirectionsResponseFactoriesKt#createIncident`

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/6369